utils/exceptions.cpp

```diff
@@ -0,0 +1,50 @@
+#include "utils/exceptions.hpp"
+#include <QTextStream>
+
+namespace jbrush {
+
+// GENERAL EXCEPTION DEFS
+GeneralException::GeneralException() {}
```

Comment:
Why the default constructors for all of the exception classes?

Reply:
Base class needs it because some of the derived class constructors don't have a message to initialize it with, but I guess the derived classes don't need defaults

Reply:
And if the derived don't need defaults, then the base doesn't need it.